### ISSUE TITLE
"Refresh" button present in "Saved Connections" does not have name defined: A11y_mssql for VSCode_Add Connection_Name Role Value

### ISSUE THREAD

**Issue by ManishaC1** [microsoft/vscode-mssql#18653]
**Github Tags:** #A11yTCS; #A11ySev2; #A11yMAS; #WCAG4.1.2; #Name Role Value; #FTP; #Win32; #Win11; #DesktopApp; #MSSQL for VSCode; #BM_MSSQL for VSCode_Win32_Feb2025; #AILimited; #Rev:shta; #Closed; #A11yFixed;

**Please do not close this bug. This bug should only be closed by Trusted Tester after verification** 

"[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier."

## Environment Details:

Application Name: MSSQL for VSCode

Windows 11 24H2 OS Build 26100.3037

## Repro Steps:

1. Launch VS Code Application.
2. Open SQL Server Extension.
3. Tab to "New Connection" and press enter.
4. Tab to "Refresh" button present in "Saved Connections" section.
5. Observe accessible name is not defined.

## Actual Results:
The "Refresh" button within the "Saved Connections" section does not have a defined name.

## Expected Results:
The "Refresh" button within the "Saved Connections" section should have a defined name such as "Refresh".

## User Impact:
Screen Reader Users cannot identify the "Refresh" button, leading to a poor user experience.

## WCAG Reference:  

https://www.w3.org/WAI/WCAG22/Understanding/name-role-value

## Attachments:

![Image](https://github.com/user-attachments/assets/1081c87d-6f6b-40e1-bb39-335a309a9388)

![Image](https://github.com/user-attachments/assets/8ac82f2b-7e08-4333-ae6e-3804a14b5bf3)

**Comment by Benjin** [comment]
@ManishaC1  This should be resolved with https://github.com/microsoft/vscode-mssql/pull/19598.  Please confirm with our next release (1.34 or later)

**Comment by ManishaC1** [comment]
The 1.34 version with the necessary fixes isnt available for download yet. I’ll take action when the version is out

**Comment by ManishaC1** [comment]
Issue fixed in v1.34

<img width="1333" height="722" alt="Image" src="https://github.com/user-attachments/assets/303041a4-bdbb-49cd-a8f0-8a4c3f09d3fd" />
